### ISSUE TITLE
In the IE11 browser, when the form field is used in Chinese, it will be verified even if no operation is performed, and the verification information is prompted, and the user experience is very unfriendly.

### ISSUE THREAD

**Issue by xLong1029** [angular/angular#31904]
![TIM截图20190730160511](https://user-images.githubusercontent.com/18066675/62111744-d4dbb680-b2e3-11e9-82a4-85a905fc3fed.jpg)
### Reproduction link
[https://stackblitz.com/edit/ng-zorro-antd-start-ncko3j?file=src/app/app.component.ts](https://stackblitz.com/edit/ng-zorro-antd-start-ncko3j?file=src/app/app.component.ts)

### Steps to reproduce
Ng-alain version: 7.5.1~8.1.0
Angular version: 8.1.2
In the above version, under the browser IE11, if the form field uses Chinese characters, the following problems will occur:

After entering the project page for the first time, the user will trigger the form verification and prompt the error message without any operation.
After refreshing the page, there are the same problems, which will trigger the form verification.
If you use English, you won't have the above problems, but our users are Chinese-oriented. I personally think this is very unfriendly to the user experience. I hope I can solve it. Thank you.

### What is expected?
Under IE11 browser, after entering the project page for the first time, if it is not operated, no verification will be performed, no information will be prompted, and verification will be performed only after the user operates.

### What is actually happening?
After entering the routing page, there will be form validation and no error message if no operation is performed.

| Environment | Info |
|---|---|
| ng-zorro-antd | 8.1.2 |
| Browser | IE11 |

---
Use the login page after the project is launched

**Comment by atscott** [comment]
Thanks @trotyl - closing as duplicate of #14440, #14667, #15299, #15452, #16151, #17951, #17955.
